### DISCUSSION TITLE
Easiest way to style my stripejs form like my other tui form components

### DISCUSSION THREAD

**Discussion by ScottMGerstl** [taiga-family/taiga-ui#927] [Q&A] [answered]
Is there an example of or can someone provide guidance for a set of base components that will allow me to inherit/apply the behavior and styles of a regular tui input to a stripe js form or a custom div so I can match the rest of my form?

It's a requirement to not handle the card numbers directly so I cannot use the input card component from the tui library.

I'm using the [ngx-stripe](https://www.npmjs.com/package/ngx-stripe) library.

Using tailwind, this is where I stopped as I would have to basically reimplement the behavior taiga has so I'm looking for a better solution

```html
<div
    class="
        flex
        w-full
        mt-4
        py-2
        px-4
        rounded-lg
        bg-body-2-color
        shadow-form-field
        can-hover:hover:shadow-form-field-hover
    "
>
    <ngx-stripe-card class="w-full" [options]="cardOptions"></ngx-stripe-card>
</div>
```

**Comment by waterplea** [comment, marked as answer]
Visuals of controls is isolated in `tui-wrapper` component with `appearance="textfield"`:
https://github.com/TinkoffCreditSystems/taiga-ui/tree/main/projects/core/components/wrapper

**Reply by ScottMGerstl** [reply]
Perfect thank you.
